vanilla-framework/vanilla-framework
# Remove margin-top from stepped list content

## Done

- Removed the negative margin-top from `.p-list-step__content` which was causing alignment issues when the content was not a `<p>` (e.g a button or code snippet)

## QA

- Pull code
- Run `./run serve --watch`
- Open http://0.0.0.0:8101/vanilla-framework/examples/patterns/lists/lists-stepped-detailed/
- Check that there is no margin-top on any `.p-list-step__content`
- Replace one with:
``` html
<div class="p-list-step__content">
  <div class="p-code-snippet">
    <input class="p-code-snippet__input" value="sudo lxd init" readonly="readonly">
    <button class="p-code-snippet__action">Copy to clipboard</button>
  </div>
  <p>Text</p>
</div>
```
- Check that the alignment looks good

## Details

Fixes #1774 

## Screenshots
### Before

![screenshot_4](https://user-images.githubusercontent.com/25733845/39237912-c9bd30ec-4874-11e8-8ee9-ac6de9f8522c.png)

### After

![screenshot_5](https://user-images.githubusercontent.com/25733845/39237916-ce50678c-4874-11e8-8958-e5993e58bc52.png)
